spyder/plugins/editor/confpage.py

```diff
@@ -184,6 +193,33 @@ def setup_page(self):
             'tab_stop_width_spaces',
             default=4, min_=1, max_=8, step=1)
 
+        check_eol_box = newcb(
+            _("Fix mixed EOLs automatically and show warning"),
```

Comment:
```suggestion
            _("Fix mixed end-of-lines automatically and show warning"),
```

EOLs is a technical term that works well in English (although it's probably not known by many people). But I don't think it works in other languages (at least not in Spanish).

So, this will make translators lives easier.

Reply:
Okay, I almost made the change before but I had used "end-of-line characters" which made the item very long. Made the change you suggested plus added "dialog" after "warning" to clarify that a bit (like the original)

```diff
@@ -194,7 +194,7 @@ def setup_page(self):
             default=4, min_=1, max_=8, step=1)

         check_eol_box = newcb(
-            _("Fix mixed EOLs automatically and show warning"),
+            _("Fix mixed end-of-lines automatically and show warning dialog"),
             'check_eol_chars',
             default=True,
             tip=_(
```